src/NuGet.Clients/NuGet.CommandLine/Commands/SearchCommand.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Commands;
+
+namespace NuGet.CommandLine
+{
+    [Command(
+        typeof(NuGetCommand),
+        "search",
+        "SearchCommandDescription",
+        UsageSummaryResourceName = "SearchCommandUsageSummary",
+        UsageDescriptionResourceName = "SearchCommandUsageDescription",
+        UsageExampleResourceName = "SearchCommandUsageExamples")]
+    public class SearchCommand : Command
+    {
+        private readonly List<string> _sources = new List<string>();
+
+        [Option(typeof(NuGetCommand), "ListCommandSourceDescription")]
+        public ICollection<string> Source
+        {
+            get { return _sources; }
+        }
+
+        [Option(typeof(NuGetCommand), "ListCommandVerboseListDescription")]
+        public bool Verbose { get; set; }
+
+        [Option(typeof(NuGetCommand), "ListCommandAllVersionsDescription")]
+        public bool AllVersions { get; set; }
+
+        [Option(typeof(NuGetCommand), "ListCommandPrerelease")]
+        public bool Prerelease { get; set; }
+
+        [Option(typeof(NuGetCommand), "ListCommandIncludeDelisted")]
+        public bool IncludeDelisted { get; set; }
```

Comment:
>IncludeDelisted [](start = 20, length = 15)

The goal is to rebuild the `nuget search` experience on top of V3 APIs, right? If so, the V3 search service cannot return unlisted packages. I would remove this option.

Reply:
I agree with @loic-sharma 

The issue also says that we should remove this option. 

You have 2 options. 

1. Do it in the same PR, if it's a simple change. 
2. Create a new follow up issue and make sure it's addressed in the same release.

Reply:
Removed in another PR (not ready for review): https://github.com/NuGet/NuGet.Client/pull/2834/files#diff-07e95d5eefa54e4d7ae3d0955590dd06R19